staging/src/k8s.io/cli-runtime/pkg/resource/helper.go

```diff
@@ -152,21 +152,24 @@ func (m *Helper) Patch(namespace, name string, pt types.PatchType, data []byte,
 		Get()
 }
 
-func (m *Helper) Replace(namespace, name string, overwrite bool, obj runtime.Object) (runtime.Object, error) {
+func (m *Helper) Replace(namespace, name string, overwrite bool, obj runtime.Object, options *metav1.DeleteOptions) (runtime.Object, error) {
```

Comment:
Anything that needs to do replace is going to be impossible to dry-run server (because it's a two step process, and we can only dry-run one thing at a time). Unless I'm missing something?

Reply:
OK I was wrong here, replace is just doing an update, this is fine.